User.java

```diff
@@ -0,0 +1,76 @@
+
+public class User {
+	/*For the User class the fields needed are:
+	 * Name, age, city, state, disabilities, and
+	 * number of families.
+     * 
+     * Class also contains multiple methods for creating and sending the
+     * information of the user
+     * 
+	 * */
+	private String name;
+	private int age;
+	private String city;
+	private String state;
+	private String disabilities;
+    private int numOfFamMembers;
```

Comment:
Just adding this here for convenience:

Name: String
Age: Integer
Number of Family Members: Integer
Disabilities: Boolean
Do you need medical assistance: Boolean
How long do you need shelter: Integer

Once that and the methods are updated, it'll try to approve it ASAP 👍